### PR TITLE
update display names

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26473

## Description
Update display names for the integrations to include the company name

### CODE REVIEW

File: Packs/Remedy_AR/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "Remedy AR",
```
Comment (bakatzir) [inline Packs/Remedy_AR/pack_metadata.json:2]: just add BMC here

Comment (ronykoz) [reply Packs/Remedy_AR/pack_metadata.json:2]: i think that we will need to do another cleanup if i do it, it's in the description so i think there is no need

**Comment by ronykoz** [comment]
@amshamah419 could you go over my rn please?

File: Packs/Remedy_AR/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Remedy AR
+  - Update the name of the integration to include the company name.
```
Comment (amshamah419) [inline Packs/Remedy_AR/ReleaseNotes/1_0_1.md:4]:
```suggestion
  - Updated the name of the integration to include the company name.
```

File: Packs/remedy_SR/Integrations/remedy_SR/remedy_SR.py [hunk 3]
```diff
@@ -390,7 +390,7 @@ def remedy_create_ticket_command():
 
 
 def remedy_update_ticket_command():
-    demisto.log('TODO')
```
Comment (amshamah419) [inline Packs/remedy_SR/Integrations/remedy_SR/remedy_SR.py:393]: How did that make it into prod? lol

File: Packs/remedy_SR/ReleaseNotes/1_0_1.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### remedy_sr_beta
+  - Update the name of the integration to include the company name.
```
Comment (amshamah419) [inline Packs/remedy_SR/ReleaseNotes/1_0_1.md:4]:
```suggestion
  - Updated the name of the integration to include the company name.
```

File: Packs/remedy_SR/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### remedy_sr_beta
```
Comment (amshamah419) [inline Packs/remedy_SR/ReleaseNotes/1_0_1.md:3]: What would it take to change the name of the integration?

**Comment by amshamah419** [comment]
@ronykoz Done, just commit the changes.

File: Packs/remedy_SR/ReleaseNotes/1_0_1.md (hunk 5 above)
Comment (ronykoz) [reply Packs/remedy_SR/ReleaseNotes/1_0_1.md:3]: Not possible at the moment

**Pull request merged**